webservices/common/models/itemized.py

```diff
@@ -26,6 +30,30 @@ def memoed_subtotal(self):
         return self.memo_code == 'X'
 
 
+class BaseRawItemized(db.Model):
```

Comment:
Is this different enough from the other `BaseItemized` class that it made more sense to create a new one?  Are there a bunch of different fields or just a handful of additional fields from the other?

Is there anticipation for other e-filing records to model in the future?  Would it be possible to inherit from the `BaseItemized` class in this one and go from there?